tabcmd/commands/user/create_users_command.py

```diff
@@ -50,18 +49,11 @@ def run_command(args):
                 # TODO: bring in other attributes in file, actually act on specific site
                 new_user = TSC.UserItem(user_obj.name, args.role)
                 result = server.users.add(new_user)
-                print(result)
                 logger.info("Successfully created user: {}".format(user_obj.name))
                 number_of_users_added += 1
             except TSC.ServerResponseError as e:
```

Comment:
nit: Should we call the common error check instead of the specific TSC ServerResponseError here? 

` except Exception as e:
           Errors.check_common_error_codes_and_explain(logger, e)
`

Reply:
yes - fixed